libs/components/indicators/src/lib/modules/wait/wait.service.spec.ts

```diff
@@ -76,9 +81,7 @@ describe('Wait service', () => {
     TestBed.configureTestingModule({
       imports: [SkyWaitFixturesModule],
     });
-  });
 
-  beforeEach(() => {
```

Comment:
We had 2 `beforeEach` calls for some reason - it works with them combined so I just cleaned it up.